sources/packages/backend/apps/api/src/route-controllers/student-loan-balance/models/student-loan-balance.dto.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Student loan balance details of a month.
```

Comment:
As a suggestion, the word monthly can be misleading and I am not sure if adds some value. The report is received monthly but the balance "is just a balance" that represents the lastest state.